freeipa/src/main/java/com/sequenceiq/freeipa/service/FreeIpaPostInstallService.java

```diff
@@ -24,5 +28,9 @@ public void postInstallFreeIpa(Long stackId) throws Exception {
         FreeIpaClient freeIpaClient = freeIpaClientFactory.getFreeIpaClientForStackId(stackId);
         freeIpaClient.addPasswordExpirationPermission(SET_PASSWORD_EXPIRATION_PERMISSION);
         freeIpaClient.addPermissionToPrivilege(USER_ADMIN_PRIVILEGE, SET_PASSWORD_EXPIRATION_PERMISSION);
+        if (!Objects.equals(MAX_USERNAME_LENGTH, freeIpaClient.getUsernameLength())) {
```

Comment:
Currently both arguments are primitive `int`s, so not justifying the usage of `Objects.equals()`. On the other hand, as per the above comment, `FreeIpaClient.getUsernameLength()` was probably supposed to return an `Integer`.